tsconfig.json

```diff
@@ -15,8 +15,8 @@
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     "sourceMap": true,                        /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "dist",                        /* Redirect output structure to the directory. */
-    "rootDir": ".",                         /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    "outDir": "dist",                         /* Redirect output structure to the directory. */
+    "rootDir": "src",                         /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

Comment:
This makes the `dist/` like:
```
dist/index.js
dist/...
```

Previously, it was:
```
dist/src/index.js
dist/src/...
dist/test/...
```
which is not compatible with our `package.json`.